src/scraping-defaults.ts

```diff
@@ -21,6 +21,8 @@ const SCRAPING_DEFAULT_OPTIONS = {
     },
 };
 
+export { SCRAPING_DEFAULT_OPTIONS };
```

Comment:
Thats..not what I meant :P

I meant adding the `export` keyword in front of `const SCRAPING...` 😝

Reply:
ah yea I didn't notice it, it's so obv :P